src/utils.js

```diff
@@ -31,8 +31,13 @@ export const loadScript = function (id) {
     return
   }
 
+  let querys = []
+  for (const [key, value] of Object.entries(queryParams)) {
+    querys.push(key + '=' + value)
+  }
+
```

Comment:
Thanks for your PR, I also looking for the solution to pass extra variables into config. 

There is could be more graceful solution 
```
const searchParams = new URLSearchParams(queryParams);

searchParams.append('id', id);
script.src   = `https://www.googletagmanager.com/gtm.js?{searchParams.toString()}`
```

Reply:
Oh, I think it's more graceful implementation let the code more readable
Thanks for your suggestion!